client/src/main/java/org/apache/rocketmq/client/impl/factory/MQClientInstance.java

```diff
@@ -255,19 +255,17 @@ public void start() throws MQClientException {
     }
 
     private void startScheduledTask() {
-        if (null == this.clientConfig.getNamesrvAddr()) {
```

Comment:
@shroman 

while to me, I will consider that if the nameserver address is specified manually, no dramatically fetch should be triggered since we should respect what user's choice. Or they may be confusing about the inconsistent behavor.

So, I would think the existing design for:

1. when specified nameserver address, no need to fetch nameserver address dymatically.
2. Only fetch nameserver address when user has not specified manully.